example/android/app/build.gradle

```diff
@@ -26,7 +26,7 @@ apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 30
+    compileSdkVersion 31
```

Comment:
This is really nice, thank you for updating the example, just one little problem though. This would stop the project from running unless you add the `android:exported="true"` to [this activity](https://github.com/garlicPasta/swipe_image_gallery/blob/master/example/android/app/src/main/AndroidManifest.xml#L6).

Reply:
Yes true. Fixed it. ✔️